tetrateci/fetch_all_tags.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+git remote add upstream https://github.com/istio/istio
+git fetch --tags origin
+git tag -l | grep -E "^[0-9]+.[0-9]+.[0-9]+$" > f1
+git fetch --tags upstream
+git tag -l | grep -E "^[0-9]+.[0-9]+.[0-9]+$" > f2
+
+echo "print f1"
+cat f1
+echo "print f2"
+cat f2
+
+tags=$(comm -13 f1 f2)
+rm f1 f2
+
+git config user.name github-actions
+git config user.email github-actions@github.com
+
+for tag in $tags; do
+    branch=$( echo $tag | rev | cut -d. -f2- | rev )
```

Comment:
won't this do. -- echo $tag | cut -d. -f1,2?

Reply:
Ahh right, the tags have a fixed shape